cookbooks/cdo-ruby/metadata.rb

```diff
@@ -1,9 +1,10 @@
-name             'cdo-ruby-2.0'
+name             'cdo-ruby'
 maintainer       'Code.org'
-maintainer_email 'geoffrey@code.org'
+maintainer_email 'will@code.org'
 license          'All rights reserved'
-description      'Installs/Configures cdo-ruby-2.0'
+description      'Installs/Upgrades Ruby'
```

Comment:
Added [README.md](https://github.com/code-dot-org/code-dot-org/blob/cdo-ruby/cookbooks/cdo-ruby/README.md) with this detailed info on available attributes.